src/cli/README_COMMISSIONER.md

```diff
@@ -120,6 +121,24 @@ Done
 Done
 ```
 
+### joiner table
+
+Usage: `commissioner joiner table`
+
+List all Joiner entries.
+
+```bash
+> commissioner joiner table
+
```

Comment:
According to code, there should be no preleading empty line ?

Reply:
Yes, it was a copy & paste mistake. Removed.